README.md

```diff
@@ -9,6 +9,30 @@ An abstraction layer for low-level platform-specific graphics APIs.
 * Reference documentation: http://wiki.gpfault.net/docs/nicegraf
 * Sample code: https://github.com/nicebyte/nicegraf/tree/master/samples
 
+# prerequisites
+
+## Windows
+
+- CMake 3.14.0 or above
+- Visual Studio 2022 with v143 build tools (found in Desktop Development with C++ section)
```

Comment:
Remove the specific IDE / build tools version. All we need is a C99-capable C compiler and a C++14-capable compiler.

As a consequence, remove "Windows" from the prerequisites list,  it is now platform agnostic.